plugins/linux-pipewire/linux-pipewire.c

```diff
@@ -34,6 +35,10 @@ MODULE_EXPORT const char *obs_module_description(void)
 
 bool obs_module_load(void)
 {
+	obs_enter_graphics();
+	gladLoadGL();
```

Comment:
I take that it was easier to just load all symbols vs loading specific ones?

Reply:
Correct its easier and less error prone to keep glad actually initialized.